### ISSUE TITLE
SCOM.xsl:129-131: Calculate SCOM_min as per page 86 of...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/jpeek#142]
The puzzle `68-03d5665d` from #68 has to be resolved:

https://github.com/yegor256/jpeek/blob/aae0a5fa3342477fe23137e583903ce03c20af79/src/main/resources/org/jpeek/metrics/SCOM.xsl#L129-L131

The puzzle was created by George Aristy on 30-Jan-18. 

Estimate: 30 minutes, role: IMP.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#142](https://github.com/yegor256/jpeek/issues/142) is now in scope, role is `DEV`

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0crat** [comment]
The job [#142](https://github.com/yegor256/jpeek/issues/142) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/540904269). The budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by llorllale** [comment]
@0crat waiting on an answer to [this](https://math.stackexchange.com/questions/2667542/understanding-int-and-mod-functions-as-used-in-this-formula) question on math.stackexchange.com in order to be sure on what the `mod()` function means

**Comment by 0crat** [comment]
> @0crat waiting on an answer to [this](https://math.stackexchange.com/questions/2667542/understand... ([here](https://github.com/yegor256/jpeek/issues/142#issuecomment-368530038))

@llorllale The impediment for [#142](https://github.com/yegor256/jpeek/issues/142) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@yegor256 I'm using [this](https://math.stackexchange.com/a/2667550/534583)  answer as the definition of the `mod()` function

**Comment by llorllale** [comment]
@yegor256 `MetricsTest` currently does not provide a way to test this. What should I do in that regard?

**Comment by yegor256** [comment]
@llorllale create another ticket, I believe. Did you read this? http://www.yegor256.com/2015/01/15/how-to-cut-corners.html

**Comment by llorllale** [comment]
@yegor256 we already have an open issue on `MetricsTest` in #169 but I am not sure if I can declare it as an impediment since it's still not in scope. The proposal for #169 would allow me to add this simple test.

**Comment by llorllale** [comment]
@yegor256 fine, I created #222, I hope it's ok

**Comment by llorllale** [comment]
@0crat waiting on review for PR #223

**Comment by 0crat** [comment]
> @0crat waiting on review for PR #223  ([here](https://github.com/yegor256/jpeek/issues/142#issuecomment-370426248))

@llorllale You can't do that, unless you have one of these roles: `ARC`, `PO`. Your current roles are: `DEV`.

**Comment by 0crat** [comment]
> @0crat waiting on review for PR #223  ([here](https://github.com/yegor256/jpeek/issues/142#issuecomment-370426248))

@llorllale Job [#142](https://github.com/yegor256/jpeek/issues/142) is already on hold

**Comment by 0pdd** [comment]
The puzzle `68-03d5665d` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
The job [#142](https://github.com/yegor256/jpeek/issues/142) is now out of scope
